modules/apps/site/site-cms-site-initializer/test/js/main_view/dashboard/RangeSelectorsDropdown.test.tsx

```diff
@@ -3,16 +3,16 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import '@testing-library/jest-dom/extend-expect';
-import {fireEvent, render, screen} from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
-import React from 'react';
-
 import {
 	IRangeSelectorsDropdown,
 	RangeSelectors,
 	RangeSelectorsDropdown,
-} from '../../../../src/main/resources/META-INF/resources/js/main_view/dashboard/components/RangeSelectorsDropdown';
+} from '@liferay/analytics-reports-js-components-web';
+
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Could you move this import to be the first one? And then sort the other ones?

Reply:
done!